Keen/AccessKey/AccessKey.cs

```diff
@@ -0,0 +1,79 @@
+﻿using Keen.Core.Query;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using Newtonsoft.Json.Serialization;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Keen.Core.AccessKey
+{
+    /// <summary>
+    /// Modal for AccessKey object
```

Comment:
Typo: "Modal"->"Model", and we should elaborate on XML doc comments on any public members.